Source/Core/OrientedBoundingBox.js

```diff
@@ -419,6 +419,34 @@ define([
         return result;
     };
 
+    /**
+     * @private
+     */
+    OrientedBoundingBox.closestLocationIn = function(position, boundingObject) {
+        if (defined(boundingObject)) {
+            // to avoid inverse of a zero matrix (since that doesn't exist).
+            if (this.boundingObject.halfAxes.equals(Matrix3.ZERO)) {
+                position = new Cartesian3();
+                return position;
+            }
+
+            // convert world space positionToCheck to orientedBoundingBox's object space.
+            var inverseTransformationMatrix = Matrix3.inverse(boundingObject.halfAxes, new Matrix3.fromScale(1.0));
+            var positionInObjectSpace = Matrix3.multiplyByVector(inverseTransformationMatrix, position, new Matrix3.fromScale(1.0));
+
+            // once in object space just check if converted location is within axis oriented 2x2x2 cube (bc half axes)
+            var minimum = new Cartesian3(-1.0, -1.0, -1.0);
+            var maximum = new Cartesian3(1.0, 1.0, 1.0);
+
+            positionInObjectSpace.x = CesiumMath.clamp(positionInObjectSpace.x, minimum.x, maximum.x);
```

Comment:
Same comment as for `AxisAlignedBoundingBox`.  `boundingObject.center` should be used somewhere in this function.